.github/workflows/snyk.yml

```
@@ -0,0 +1,63 @@
+name: "Snyk"
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  DEFAULT_JDK_VERSION: 11
+
+jobs:
+  quarkus:
+    name: Quarkus
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v2
+        with:
+          java-version: ${{ env.DEFAULT_JDK_VERSION }}
```

Not entirely sure but I believe you need to specify a `distribution` property here.
```suggestion
          java-version: ${{ env.DEFAULT_JDK_VERSION }}
          distribution: zulu
```